_features/installable.html

```diff
@@ -2,8 +2,17 @@
 title: Installable
 slug: installable
 description: Keep user's most useful app on their home screen without the hassle of an app store.
+usecases:
+- As a user, I want my favourite web applications (GMail, YouTube,
+  Facebook&hellip;) to appear in my homescree as if they were been installed from
+  the marketplace.
+- As a browser developer, I want to offer the users the posibility of provide a
+  native user experience for a web site.
```

Comment:
Nits:
* posibility -> possibility
* of provide -> to provide
* experience for -> experience of
* web site -> web app

But it isn't the users who provide the experience. so I'd make this, "As a browser developer, I want to offer users the possibility to have a native user experience of a web site."